SQL/Cleanup_patches/README.md

```diff
@@ -0,0 +1 @@
+This directory contains SQL patches written for post-release cleanup. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements. These patches should be run at the discretion of the projects with the full knowledge of the possible side effects of these patches. The content of these patches should not be mandotory for proper functioning of the database, they should be run for the sole purpose of removing unused tables, columns or fields; any mandatory cleanup should be added to the release patches.
```

Comment:
You mention DELETE statement but near the end of the paragraph, you mention removing columns or fields, not rows.

Could you add precision about when DELETE could be used? or remove DELETE statement?

Reply:
rows should be there as well, adding it to the  readme